components/ord-service/pom.xml

```diff
@@ -18,7 +18,7 @@
         <project.build.sourceEncoding>UTF-8</project.build.sourceEncoding>
         <project.build.source>1.8</project.build.source>
         <java.version>11</java.version>
-        <jpa-processor>0.3.9-SNAPSHOT</jpa-processor>
+        <jpa-processor>0.3.11.2-SNAPSHOT</jpa-processor>
```

Comment:
Out of curiosity - why do we bump this version? Is there something useful that is used in the fork?

Reply:
Regarding the bump from 3.9 to 3.11 I saw some bugfixes that are introduced in 3.10 and 3.11 and explicitly stated that it is not a breaking change, therefore I decided to do it as I am working on that.

Regarding the 4th number. I need to do a tag in the fork, and I cannot directly change an existing tag, therefore I decided that it is a good pattern to add one more number that tracks how many custom patches we have done to a specific version.

This means that we are using 0.3.11 from the original repo and have two patches from our self in the form -> 0.3.11.2

Reply:
Makes sense, thanks 👍